### PR TITLE
ffac-wg-registration: fix all shellcheck issues

### PR DESCRIPTION
**Pull request by grische**

### CODE REVIEW

File: ffac-wg-registration/files/lib/gluon/wg-registration/registration.sh [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/bin/sh
 
-if [ "$(uci get gluon.mesh_vpn.enabled)" == "true" ] || [ "$(uci get gluon.mesh_vpn.enabled)" == "1" ]; then
+if [ "$(uci get gluon.mesh_vpn.enabled)" = "true" ] || [ "$(uci get gluon.mesh_vpn.enabled)" = "1" ]; then
         # check if registration has been done since last boot
         if [ ! -f /tmp/WG_REGISTRATION_SUCCESSFUL ]; then
 		# Push public key to broker, test for https and use if supported
-		wget -q https://[::1]
+		wget -q "https://[::1]"
```
Comment (maurerle) [inline ffac-wg-registration/files/lib/gluon/wg-registration/registration.sh:7]: Shouldn't this also be `if ! wget -q "https://[::1]"; then` ?

Comment (grische) [reply ffac-wg-registration/files/lib/gluon/wg-registration/registration.sh:7]: Absolutely, thanks for spotting this

Comment (grische) [reply ffac-wg-registration/files/lib/gluon/wg-registration/registration.sh:7]: I think I found an issue in this code actually. I will open a separate PR to address those

Comment (grische) [reply ffac-wg-registration/files/lib/gluon/wg-registration/registration.sh:7]: See https://github.com/freifunk-gluon/community-packages/pull/58 for related discussion

**Pull request merged**